app/Stacks.js

```diff
@@ -6,6 +6,7 @@ const OFFSET_LEFT = 35;
 const OFFSET_TOP = 15;
 const uuidv4 = require('uuid/v4');
 const Card = require('../app/Card.js');
+const Canvas = require('../app/Canvas.js')
```

Comment:
Allowing the `Stack` class to be aware of the `Canvas` class inverts the structure of the application as defined and enabled by the use of object-oriented design principles. It is not obvious why this inclusion is even necessary in the current version of `Stack` as defined below. Please remove and refactor as needed.